test/ObjectField_test.js

```diff
@@ -50,7 +50,10 @@ describe("ObjectField", () => {
     });
 
     it("should render a customized title", () => {
-      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+      const CustomTitleField = ({title, required}) => {
+        const legend = required ? title + '*' : title;
+        return <div id="custom">{legend}</div>;
+      };
```

Comment:
Is this really necessary as we don't have any related assertion here? Should we add one?